app/helpers/application_helper.rb

```diff
@@ -1,5 +1,11 @@
+include MarkdownContent
+
 module ApplicationHelper
   def theme
     League.current.theme || "default"
   end
+
+  def markdown(text)
+    MarkdownContent.markdown(text.to_s).html_safe
+  end
```

Comment:
If you `include MarkdownContent` inside the helper, then it exposes that functionality within this file and this alias-y method is not required.
```
module ApplicationHelper
  include MarkdownContent

  ...
end
```

The only thing that remains is the `html_safe` call, but we can make it consistent by using `raw` in the view like I have elsewhere. (e.g. `app/views/rules/show.html.haml`)
```
= raw markdown(whatever)
```
Now I'm not 100% sure that's the best way to do it, but it is at least consistent. There's an argument for refactoring the `raw` call into the `MarkdownContent` concern actually and removing it from all these views. Fancy giving that a try?